libraries/ms-helper/src/thermistor_converter.c

```diff
@@ -0,0 +1,52 @@
+#include "thermistor_converter.h"
+#include <stdint.h>
+#include "log.h"
+
+// resistance to milliohms
+uint32_t resistance[RESISTANCE_LEN] = {
```

Comment:
Resistance to milliohms?

Please make sure this follows the naming scheme. If this is a lookup, please document where this data came from and what it represents. You should also make it static const since it should not be changing and should not be accessed outside of this module.